llmfoundry/utils/exceptions.py

```diff
@@ -524,3 +525,11 @@ def __reduce__(self):
 
     def __str__(self):
         return self.message
+
+
+class ProbablyNetworkingError(UserError):
```

Comment:
should we just raise this as a permissions error? In the finetuning API, we can add the messaging around PL/SEG, since we know that is the cause there, but in general you could hit this for a variety of permissions error so we might want to keep the foundry message more general? wdyt

Reply:
This is a naming thing right? Rename this `ProbablyNetworkingError`->`StoragePermissionsError`.

That sounds fine to me.

Reply:
makes sense to me, and yes, i just meant naming (and probably adjust the message a bit)